src/client.js

```diff
@@ -353,6 +353,10 @@ export function MatrixClient(opts) {
     if (call) {
         this._callEventHandler = new CallEventHandler(this);
         this._supportsVoip = true;
+        // Start listening for calls after the initial sync is done
+        // We do not need to backfill the call event buffer
+        // with encrypted events that might never get decrypted
```

Comment:
I think we do want to start listening before the catchup sync to catch calls placed shortly before the user opened their client, but its probably safe to start listening after the events are loaded from the database, so after the state changes to PREPARED I think?